apps/challenges/migrations/0006_upload_to_attrbute_added.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.2 on 2016-12-24 10:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('challenges', '0005_changed_phase_model'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='challenge',
+            name='evaluation_script',
+            field=models.FileField(default=False, upload_to='evaluation_scripts'),
```

Comment:
@deshraj : forgot to update the migrations. can you please update both folder names here?